javatests/google/registry/rde/imports/RdeDomainImportActionTest.java

```diff
@@ -174,6 +331,14 @@ private void pushToGcs(ByteSource source) throws IOException {
     return result;
   }
 
+  private static BillingEvent.Recurring loadAutorenewBillingEventForDomain(DomainResource domain) {
```

Comment:
I'm wondering if these should be annotated with @Nullable, because ofy().load() will return null if the entity in question doesn't exist.  Now granted, it should always exist ... but it's more likely not to if someone failed in an import mapreduce, versus a standard transactional EPP flow, for which it's really hard to imagine such an occurrence ever happening.